user-story-template.md

```diff
@@ -0,0 +1,11 @@
+### Title:
+- Priority:
+- Estimate:
+#### Details:
+- As a <type_of_user>
```

Comment:
If we had user personas, we could totally link to those from here. Maybe when we're close to wrapping up and grooming these user stories for technical accuracy we can go and backfill those. 🙃